src/js/vre/chapter36/config/form.js

```diff
@@ -85,6 +86,61 @@ const applicantServiceHistory = _.merge(serviceHistory, {
   }
 });
 
+const previousBenefitApplicationsUI = {
+  'ui:title': 'Have you ever applied for any of the following VA benefits? (Check all that apply.)',
+  'ui:options': {
+    showFieldLabel: true
+  },
+  chapter31: {
+    'ui:title': benefitsLabels.chapter31,
+    'ui:options': {
+      hideIf: ({ 'view:isVeteran': isVeteran }) => !isVeteran
```

Comment:
Could we make two functions, `isVeteran` and `isNotVeteran`, and use those everywhere? Feel like it would be clearer to be consistent in how we use those.

Reply:
Thanks @jbalboni, good idea, changed.